opel-agent/values.yaml

```diff
@@ -49,7 +51,15 @@ opentelemetry-collector:
             - otlp
             - zipkin
             - jaeger
-        metrics: null
+        metrics:
+          exporters:
+            - coralogix
+          processors:
+            - memory_limiter
+            - batch
+          receivers:
+            - prometheus # The Prometheus receiver is configured by default
```

Comment:
@avivgold098 added this comment or, or do you think it should be in another place?